packages/spartan/extra/dcos-spartan.service

```diff
@@ -19,6 +19,6 @@ ExecStartPre=/usr/bin/env ip link set spartan up
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.1/32 dev spartan
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.2/32 dev spartan
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.3/32 dev spartan
-ExecStart=/opt/mesosphere/active/spartan/spartan/bin/spartan-env foreground
+ExecStart=/usr/bin/env flock -x /opt/mesosphere/active/spartan/spartan/bin/spartan-env /opt/mesosphere/active/spartan/spartan/bin/spartan-env foreground
```

Comment:
The other thing (other than flock) we can do is script around systemd, and ask systemd is spartan is up before running the watchdog. 

We can actually change the dcos-spartan-watchdog.service, and just move the flock check into ExecStartPre, but not for dcos-spartan.